department-of-veterans-affairs/va.gov-cms
# VACMS-2503: Health services dashboard on facility page

## Description

See #2503. 

## Testing done

Manual, some behat spec tool 

## Screenshots

![VA_Pittsburgh_Health_Care___H__John_Heinz_III_Department_of_Veterans_A](https://user-images.githubusercontent.com/643678/89204662-785d4800-d584-11ea-86fb-4fb6822fc270.jpg)


## QA steps

1. Go to admin/content
1. Filter by VAMC facility content type and VISN 4
1. Visit a random sample of VAMC facilities 
  - [x] Check minor improvements to proofing experience, with new fieldsets added. (This proofing UX will come later)
1. Check the health services data is filtered correctly (you can edit a couple to make sure it's tied to the right facility)
